<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="hlImport">
      <item>
       <widget class="QLabel" name="lblFileName">
        <property name="text">
         <string>Имя файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFileName"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelectFileName">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>Импорт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlExport">
      <item>
       <widget class="QLabel" name="lblExport">
        <property name="text">
         <string>Имя файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtExportFileName"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelectExportFileName">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>Экспорт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Сессия</string>
    </property>
    <addaction name="actLogin"/>
    <addaction name="actLogout"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actConnection"/>
    <addaction name="actDefaults"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="actAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actLogin">
   <property name="text">
    <string>Подключиться к базе данных</string>
   </property>
  </action>
  <action name="actLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отключиться от базы данных</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actConnection">
   <property name="text">
    <string>База данных</string>
   </property>
  </action>
  <action name="actDecor">
   <property name="text">
    <string>Внешний вид</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="actDefaults">
   <property name="text">
    <string>Умолчания</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
